src/Microsoft.AspNet.OData.Shared/Query/Validators/FilterQueryValidator.cs

```diff
@@ -213,6 +213,28 @@ public virtual void ValidateBinaryOperatorNode(BinaryOperatorNode binaryOperator
             }
         }
 
+        /// <summary>
+        /// Override this method to restrict the 'in' query inside the filter query.
+        /// </summary>
+        /// <remarks>
+        /// This method is intended to be called from method overrides in subclasses. This method also supports unit-testing scenarios and is not intended to be called from user code.
+        /// Call the Validate method to validate a <see cref="FilterQueryOption"/> instance.
+        /// </remarks>
+        /// <param name="inNode">InNode to validate</param>
+        /// <param name="settings">Settings to check against</param>
+        public virtual void ValidateInNode(InNode inNode, ODataValidationSettings settings)
```

Comment:
@mikepizzo @xuzhg @biaol-odata 
Forgot to bring this up during code review. Are there any settings that we need to apply to the InNode?

Reply:
Not that I can think of...

---
In reply to: [198664623](https://github.com/OData/WebApi/pull/1497#discussion_r198664623) [](ancestors = 198664623)

Reply:
Thanks, I'll remove this part of the code.